### DISCUSSION TITLE
Are the commands in zephyr/samples/drivers/adc/README.rst correct?

### DISCUSSION THREAD

**Discussion by MogoBagginz** [zephyrproject-rtos/zephyr#57805] [Q&A] [answered]
Hi, I was wondering why the commands in zephyr/samples/drivers/adc/README.rst were different to the one here:
https://docs.zephyrproject.org/latest/samples/drivers/adc/README.html

where the first states that these commands should be used 

".. zephyr-app-commands::
   :zephyr-app: samples/drivers/adc
   :board: nucleo_l073rz
   :goals: build flash
   :compact:
"

and the second uses this instead:

"west build -b nucleo_l073rz samples/drivers/adc
west flash"

I couldn't understand how to use the first set of commands, so I thought that maybe they were out of date?

**Comment by yashi** [comment, marked as answer]
The first one you quoted is in rst source form.  That's how you write in the source file, README.rst.  Sphinx or the doc build system converts the `.. zephyr-app-commands::` block to the one displayed in the actual document, which starts with `west`.
